securesystemslib/keys.py

```diff
@@ -687,7 +687,7 @@ def create_signature(key_dict, data):
   securesystemslib.formats.ANYKEY_SCHEMA.check_match(key_dict)
 
   # Signing the 'data' object requires a private key. Signing schemes that are
-  # currently supported are: 'ed25519', 'ecdsa-sha2-nistp256', and rsa schemes
+  # currently supported are: 'ed25519', 'ecdsa', and rsa schemes
```

Comment:
If I'm not misreading this, we're talking about the schemes themselves here, right? Meaning we leave it as is / include `ecdsa-sha2-nistp384`?

Reply:
Well spotted, thanks!